solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/Cell/RepositoryViewCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RepositoryViewCell.swift
+//  GitHubApp
+//
+//  Created by Felipe on 06/07/22.
+//
+
+import UIKit
+
+final class RepositoryViewCell: UITableViewCell {
+    
+    struct RepositoryViewCellConfiguration{
+        let repositoryName: String
+        let repsitoryOwnerName: String
```

Comment:
Como essas duas propriedades já fazem parte do escopo de Repository, usar o pronome dentro dos campos, se torna redundante, podemos reduzir utilizando somente o que eles representam dentro de `RepositoryViewCellConfiguration`
```suggestion
        let name: String
        let ownerName: String
```